client/src/templates/Introduction/components/block.tsx

```diff
@@ -118,21 +114,15 @@ class Block extends Component<BlockProps> {
     });
 
     const isProjectBlock = challenges.some(({ challenge }) => {
-      const isTakeHomeProject = blockDashedName === 'take-home-projects';
```

Comment:
Take home projects will no longer be collapsable. I think this is a decent sacrifice to reduce the bloat.

Reply:
I wouldn't change behaviour in a refactor. It makes it harder to review and track where/why a change was made.